helper/helper-common/src/main/java/com/linkedin/avroutil1/compatibility/AvroAdapter.java

```diff
@@ -134,10 +136,14 @@ default FieldBuilder newFieldBuilder(String name) {
 
   void setFieldPropFromJsonString(Schema.Field field, String propName, String valueAsJsonLiteral, boolean strict);
 
+  boolean sameJsonProperties(Schema.Field a, Schema.Field b, boolean compareStringProps, boolean compareNonStringProps);
```

Comment:
Maybe we could document somewhere why we support enabling/disabling checking string props and non-string props?

Reply:
will do. answer is because avro 1.4 only treats the string props, and we want to be able to replicate this behaviour under any avro